.github/workflows/rust.yml

```diff
@@ -62,11 +62,10 @@ jobs:
       uses: jetli/wasm-pack-action@v0.4.0
       with:
         version: 'latest'
-
-    - name: Run Ethereum tests
+    - name: Set environment variables
       run: |
-        cargo test -p linera-ethereum --features ethereum
-        cargo test test_wasm_end_to_end_ethereum_tracker --features ethereum
+        echo "LINERA_STORAGE_SERVICE=127.0.0.1:1235" >> "$GITHUB_ENV"
```

Comment:
Is that different from just adding

```yml
    LINERA_STORAGE_SERVICE: 127.0.0.1:1235
```

to the `env` section?

Reply:
No.

Reply:
It's a matter of taste if you want to group the numbers `1235` together or not.